packages/patricia-tree/patricia-tree.0.9.0/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis:
+  "Patricia Tree data structure in OCaml for maps and sets. Supports generic key-value pairs"
+maintainer: ["Dorian Lesbre <dorian.lesbre@cea.fr>"]
+authors: [
+  "Matthieu Lemerre <matthieu.lemerre@cea.fr>"
+  "Dorian Lesbre <dorian.lesbre@cea.fr>"
+]
+license: "LGPL-2.1-only"
+homepage: "https://codex.top/patricia-tree/"
+doc: "https://codex.top/patricia-tree/"
+bug-reports:
+  "https://github.com/codex-semantics-library/patricia-tree/issues"
+depends: [
+  "ocaml" {>= "4.14"}
+  "zarith" {>= "1.13"}
+  "dune" {>= "2.7" & >= "2.0"}
```

Comment:
```suggestion
  "dune" {>= "2.7"}
```

This should be equivalent but simpler.

Reply:
Nicely spotted, I pushed a fix. Not sure why dune outputs these double requirements when generating opam files.